lib/galaxy_test/driver/driver_util.py

```diff
@@ -794,8 +772,6 @@ def launch_server(app_factory, webapp_factory, prefix=DEFAULT_CONFIG_PREFIX, gal
     )
     if name == "galaxy":
         asgi_app = init_galaxy_fast_app(wsgi_webapp, app)
-    elif name == "tool_shed":
-        asgi_app = init_tool_shed_fast_app(wsgi_webapp, app)
     else:
         raise NotImplementedError(f"Launching {name} not implemented")
```

Comment:
What about adding `init_fast_app = init_galaxy_fast_app` as an additional parameter to `launch_server()`, instead of duplicating the function in `lib/tool_shed/test/base/driver.py` ?
Then we could just replace this block with:
```python
    asgi_app = init_fast_app(wsgi_webapp, app)
```

Reply:
Thanks, that's a good idea.